.gitignore

```diff
@@ -128,4 +128,10 @@ node_modules/
 
 # GraphViz
 *.png
-*.dot
\ No newline at end of file
+*.dot
+
+# src tarballs
+.src.tar.gz
```

Comment:
Could we relax this to `*.gz` to prevent any `GZ` additions in general?

Reply:
we could, but i do like to specify exactly which file we are ignoring